src/acom_music_box/main.py

```diff
@@ -151,13 +151,8 @@ def main():
     # Create and load a MusicBox object
     myBox = MusicBox()
     logger.debug(f"Configuration file = {musicBoxConfigFile}")
-    myBox.readConditionsFromJson(musicBoxConfigFile)
-
-    # Create solver and solve
-    config_path = os.path.join(
-        os.path.dirname(musicBoxConfigFile),
-        myBox.config_file)
-    myBox.create_solver(config_path)
+    myBox.loadJson(musicBoxConfigFile, myBox)
```

Comment:
```suggestion
    myBox.loadJson(musicBoxConfigFile)
```
See my comments on the `loadJson` method